test/testenv/testenv_test.go

```diff
@@ -0,0 +1,130 @@
+package testenv
+
+import (
+	"flag"
+	"io"
+	"log"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/pborman/uuid"
+	"golang.org/x/net/context"
+	mixerapi "istio.io/api/mixer/v1"
+	"istio.io/mixer/adapter/denier"
+	"istio.io/mixer/pkg/attribute"
+	"istio.io/mixer/pkg/handler"
+	"istio.io/mixer/template"
+)
+
+func closeHelper(c io.Closer) {
+	err := c.Close()
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func copyFile(dest, src string) error {
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+
+	defer closeHelper(in)
+	out, err := os.Create(dest)
+	if err != nil {
+		return err
+	}
+	defer closeHelper(out)
+	_, err = io.Copy(out, in)
+	return err
+
+}
+
+func buildTestConfigStore(relativePaths []string) (string, error) {
```

Comment:
`buildConfigStore`, we're already in a _test.go file so we know it has to be for a test.

Reply:
done